File: c/ege18.47v2.c
#include <stdio.h>

#define N 1024

int main() {
    int a[N];
    int n = 0;
    int k = 0;

    FILE *f = fopen("ege18.47.txt", "r");
    int x;
    while(fscanf(f, "%d", &x) != EOF) { 
        a[n] = x;
        for(int i = 0; i <= n - 11; i++) {
            if(a[i] + a[n] < 200) {
                k++;
            }
        }
        n++;
    }        

    printf("%d\n", k);
}

File: py/ege18.47v2.py
f = open("ege18.47.txt")

ax = [int(l) for l in f if l != '\n']

k = 0

for i in range(11, len(ax)):
    for j in range(i - 11 + 1):
        if ax[i] + ax[j] < 200:
            k += 1

print(k)


